src/Appwrite/Platform/Workers/Messaging.php

```diff
@@ -686,6 +686,12 @@ private function buildPushMessage(Document $message): Push
         $color = $message['data']['color'] ?? null;
         $tag = $message['data']['tag'] ?? null;
         $badge = $message['data']['badge'] ?? null;
+        $contentAvailable = $message['data']['contentAvailable'] ?? false;
+        $critical = $message['data']['critical'] ?? false;
+
+        $priority = $message['data']['priority'] === 'normal'
+            ? Priority::NORMAL
+            : Priority::HIGH;
```

Comment:
```suggestion
        $priority = $message['data']['priority'] === 'high'
            ? Priority::HIGH
            : Priority::NORMAL;
```

May be it's better to check the other way round?